tests/waveforms/cbc/test_cbc_waveforms.py

```diff
@@ -136,18 +247,83 @@ def test_taylor_f2(
         hc_ml4gw = hc_ml4gw[0]
         hp_ml4gw = hp_ml4gw[0]
 
-        assert np.allclose(
-            1e21 * hp_lal_data.real, 1e21 * hp_ml4gw.real.numpy(), atol=1e-3
-        )
-        assert np.allclose(
-            1e21 * hp_lal_data.imag, 1e21 * hp_ml4gw.imag.numpy(), atol=1e-3
-        )
-        assert np.allclose(
-            1e21 * hc_lal_data.real, 1e21 * hc_ml4gw.real.numpy(), atol=1e-3
-        )
-        assert np.allclose(
-            1e21 * hc_lal_data.imag, 1e21 * hc_ml4gw.imag.numpy(), atol=1e-3
-        )
+        if request.config.getoption("--benchmark") == 0:
+            assert np.allclose(
+                1e21 * hp_lal_data.real,
+                1e21 * hp_ml4gw.real.numpy(),
+                atol=1e-3,
+            )
+            assert np.allclose(
+                1e21 * hp_lal_data.imag,
+                1e21 * hp_ml4gw.imag.numpy(),
+                atol=1e-3,
+            )
+            assert np.allclose(
+                1e21 * hc_lal_data.real,
+                1e21 * hc_ml4gw.real.numpy(),
+                atol=1e-3,
+            )
+            assert np.allclose(
+                1e21 * hc_lal_data.imag,
+                1e21 * hc_ml4gw.imag.numpy(),
+                atol=1e-3,
+            )
+
+        # If benchmarking calculate errors to record
+        # test_2 refers to differences between lal and ml4gw
+        # TaylorF2 methods when mass ratio is flipped
```

Comment:
I suggest that we remove the benchmarking for the flipped mass ratio test. I think really we should be comparing `ml4gw` waveforms to each other for this test, not to lal, though that doesn't need to be in your PR